blip-tap.md

```diff
@@ -1091,72 +1250,109 @@ receiving node:
 - MUST reject the payment if the `tap_rfq_id` has expired based on the posted
   `expiry` value.
 
-- MUST reject the payment the `amt_to_forward != (amt_asset_incoming * tick *
-  msat_multiplier) / accepted_rate_tick
-
-where:
-
-  * `amt_asset_incoming` is the HTLC value of the `asset_id` quote accepted in
-    the earlier RFQ round
-
-  * `msat_multiplier` is a factor used to scale from `BTC/asset` to
-    `msat/asset_id`, this value is constant within the protocol and is derived
-    by multiplying the number of sats per Bitcoin, by the amount of sats in an
-    msat: `msat_multiplier = 100_000_000 * 1000 = 100_000_000_000` (100
-    billion)
-
-    * As an example: 
-      * If the incoming channel is a USD backed asset, and wishes to send $1000
-        outbound, with a rate of $61234.95 (`accepted_rate_tick = 61234.95 *
-        tick = 612_349_500` then the forwarding node expects to be instructed
-        to send `1,633,054,326 mSAT` over the outgoing link:
-          * `amt_to_forward = ($1000 * tick * msat_multiplier) // 612_349_500`
-          * `amt_to_forward = (1_000 * 10_000 * 100_000_000_000) // 612_349_500`
-          * `amt_to_forward = (1000000000000000000) // 612_349_500`
-          * `amt_to_forward = 1_633_054_326`
+- MUST reject the payment if
+  ```
+  amt_msat_to_forward > (amt_asset_incoming / incoming_asset_to_btc) * btc_to_msat_multiplier
```

Comment:
We should specify that all `*` and `/` operations are to be carried out using the `FixedPoint` abstraction. See comment above, I think we can add it as a prelimnary, then just use `*` and `/` as normal from there on.